lib/blpapi-wrapper.ts

```diff
@@ -71,15 +71,25 @@ var EVENT_TYPE = {
   , PARTIAL_RESPONSE: 'PARTIAL_RESPONSE'
 };
 
+// Map of HTTP request names to the BLAPI request and response names.
+var NAME_MAP: { [s: string]: { reqName: string; resName: string; } } = {
```

Comment:
This is a good test case for the style guide. :) @ericlu88 @ericvw what do you think of the formatting for NAME_MAP's value? It looks OK but it technically violates the current version of the style guide.

Also, we may want to consider a section on how to format type definitions.